pkgs/tools/misc/bdf2ttf/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation {
+  pname = "bdf2ttf";
+  version = "unstable-2016-07-03";
```

```suggestion
  version = "2.0-unstable-2016-07-03";
```
https://github.com/NixOS/nixpkgs/tree/master/pkgs#package-naming